ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -19,7 +19,13 @@ const FreqLink = React.memo(({ urls, value, displayValue, variant, queryParams,
     genomeVersion = variant.liftedOverGenomeVersion
   }
 
-  const path = getPath({ chrom, pos, genomeVersion, variant, value })
+  let path
```

Comment:
This would change the link for ALL populations instead of just `gnomad_svs`. You wouldn't see that bug yet since currently gnomad svs are the only frequencies with links for genomne SVs, but if we ever did add another population to these variants the links would all be wrong. `FreqLink` is created within a population-specific `FreqSummary`, and you need to make the change at that point and not here so that this change will apply only to the needed population. Currently in `FreqSummary` we use `population.af` as the `value` and `population.filter_af` as the `displayValue`. I think you should change the logic so it can use `population.id` as the value if it exists and `population.af` as the display value if need be. So inside `FreqSummary` you would change the following from what it currently is
```
const value = population.af > 0 ? population.af.toPrecision(precision) : '0.0'
const filterValue = population.filter_af > 0 ? population.filter_af.toPrecision(precision) : null
```
to 
```
const afValue = population.af > 0 ? population.af.toPrecision(precision) : '0.0'
const value = population.id ? population.id.replace('gnomAD-SV_v2.1_', '') : afValue
const displayValue = population.filter_af > 0 ? population.filter_af.toPrecision(precision) : afValue
```
Then you also need to change the implementation of `getFreqLinkPath` to use the `value` plain if its a string and not a number:
```
const floatValue = parseFloat(value, 10)
const isRegion = floatValue <= 0
let coords
if (isNaN(floatValue)) {
  coords = value
} else if (isRegion) {
  ...
```

Reply:
Sounds a good solution. I would like to learn what's different between `filter_af` and `af` when we meet next time. :)

Reply:
sure! filter_af is only populated for gnomad in the latest release. Their documentation for it is here: https://blog.limbus-medtec.com/how-to-use-gnomad-v2-1-for-variant-filtering-d7d2a7ee710a

Essentially, they create a different allele frequency calculation that takes into account subpopulations and context, which differs slightly from the uncorrected allele frequency. I don't really understand all the math and biology involved, but the summary is if there is a filter_af available its always better to filter variants based on that than on the plain af. Because of that, we want to display that value in seqr because thats what the person actually searched on. So for example, if a variant had an af of .02 and a filtering af of 0.009 and somebody applied an AF filter of .01, the variant would be returned because  the filtering af is below the threshold. But if we then showed the regular af, the person would see a variant with af of 0.02, which is above the threshhold they searched for, and think search was broken. So we show filter_Af if we have it, and then we show the plain af if you hover the frequency